addons/website_blog/models/website_blog.py

```diff
@@ -216,7 +216,7 @@ def _set_post_date(self):
         for blog_post in self:
             blog_post.published_date = blog_post.post_date
             if not blog_post.published_date:
-                blog_post._write(dict(post_date=blog_post.create_date)) # dont trigger inverse function
+                blog_post.write(dict(post_date=blog_post.create_date)) # dont trigger inverse function
```

Comment:
`_write` is not needed because it was done for method `_compute_ranking`, which doesn't exist in v16
https://github.com/odoo/odoo/commit/74e1d6940da7b5f0cc8fe4b2ba60d0bb81990ba5

`post_date` is computed method with `inverse` function which is handled in `write`, not in `_write`

https://github.com/odoo/odoo/blob/3e21dad7be482c7dbb9cd50619adf89c4b35cb8a/addons/website_blog/models/website_blog.py#L186-L187

I let someone else to check if need this patch in v15 or not :smiley:  (maybe just check if the error is reproduced before v16)

Reply:
hi,
i have checked for the same in 15 and 15.2 and couldn't reproduce it before 16.0